### PR TITLE
Add Eviction Free NYC blog post and link to site

### PR DESCRIPTION
**Pull request by romeboards**
Hello! I spoke to someone on twitter earlier about adding our [post](https://medium.com/@JustFixNYC/building-eviction-free-nyc-with-gatsbyjs-contentful-a0308bfcb866) on Medium to the blog! Hope everything checks out.

### CODE REVIEW

File: docs/blog/2018-04-27-building-eviction-free-nyc-with-gatsbyjs-and-contentful/index.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+#### **Wrapping everything up: using Netlify**
+
+When it came time to deploy, we were very excited to try out a new hosting
+platform called [Netlify](https://www.netlify.com/) that we’d been hearing a lot
+about. It has a great free tier and is perfect for hosting static sites. We
+simply linked Netlify it to a branch in the Eviction Free NYC github repo and it
+compiles and deploys the site with each push. It simplifies HTTPS certificate
+generation and even includes a system for deploying AWS Lambda functions, which
+we used to ship a small Twilio integration for the sites “Save to Phone”
```
Comment (m-allanson) [inline docs/blog/2018-04-27-building-eviction-free-nyc-with-gatsbyjs-and-contentful/index.md]: Add an apostrophe to sites? _site's_

**Comment by romeboards** [comment]
@m-allanson changes should be in place. Thanks again for the edits, we reflected them on the original post as well

**Comment by m-allanson** [comment]
Thanks @romeboards!

**Comment by jlengstorf** [comment]
Hiya @romeboards! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**